nilearn/datasets/func.py

```diff
@@ -27,12 +27,12 @@ def fetch_haxby_simple(data_dir=None, url=None, resume=True, verbose=1):
     -------
     data: sklearn.datasets.base.Bunch
         Dictionary-like object, interest attributes are:
-        'func': string.  Path to nifti file with bold data.
-        'session_target': string. Path to text file containing session and
-        target data.
-        'mask': string. Path to nifti mask file.
-        'session': string. Path to text file containing labels (can be used
-        for LeaveOneLabelOut cross validation for example).
+        - 'func': string list.  Path to nifti file with bold data.
```

Comment:
list of string rather than string list.

About adding the hyphen in front of each line, have you checked how it renders in the html API reference documentation? Personally I wouldn't bother doing that in this PR. In case there is a neat improvement in term of html rendering, you would need to do that consistently for all the fetch_* functions.